src/main/java/io/github/darealturtywurty/turtychemistry/common/block/treeblocks/rubbertree/RubberTreeLeafBlock.java

```diff
@@ -0,0 +1,12 @@
+package io.github.darealturtywurty.turtychemistry.common.block.treeblocks.rubbertree;
+
+import net.minecraft.world.level.block.Blocks;
+import net.minecraft.world.level.block.LeavesBlock;
+import net.minecraft.world.level.block.state.BlockBehaviour;
+
+public final class RubberTreeLeafBlock extends LeavesBlock {
+    //TODO:add any custom properties to this block if need be
+    public RubberTreeLeafBlock() {
+        super(BlockBehaviour.Properties.copy(Blocks.ACACIA_LEAVES));
+    }
+}
```

Comment:
Does this class really need to exist? Its not any different from a normal leaves block.

Reply:
it's a just in-case we need to apply some special properties to the rubber tree leaves, plus i'm not entire sure if a separate class is needed for it to drop saplings

Reply:
I think we can just remove this.